src/index.ts

```diff
@@ -393,8 +393,18 @@ export function apply(ctx: Context, config: Config) {
           })
 
           if (config.type === 'sd-webui') {
-            finalPrompt = (JSON.parse((res.data as StableDiffusionWebUI.Response).info)).prompt
-            return forceDataPrefix((res.data as StableDiffusionWebUI.Response).images[0])
+            const data = res.data as StableDiffusionWebUI.Response
+            if (data?.info?.prompt) {
+              finalPrompt = data.info.prompt
+            } else {
+              try {
+                finalPrompt = (JSON.parse(data.info)).prompt
+              } catch (err) {
+                const output = session.resolve(options.output ?? config.output)
+                if (output !== 'minimal') logger.warn(err)
```

Comment:
```suggestion
                logger.warn(err)
```

`output` is not relavent to logger.